typescript-test-samples/launchdarkly-lambda-dynamodb/readme.md

```diff
@@ -0,0 +1,90 @@
+# Using LaunchDarkly for Testing in AWS Lambdas
+
+## Introduction
+
+The goal of this project is to help you get started using LaunchDarkly feature flags to test new code and features within a serverless application using AWS Lambda. The project includes everything you'll need to get started integrating feature flags into your Lambdas using the Node.js runtime.
+
+The project uses the [AWS Cloud Development Kit](https://aws.amazon.com/cdk/) (CDK) for deployment. 
+
+---
+
+## Contents
+
+- [Introduction](#introduction)
+- [Contents](#contents)
+- [About this Example](#about-this-example)
+- [The LaunchDarkly SDKs](#the-launchDarkly-sdks)
+- [Deploying via the AWS CDK](#deploying-via-the-aws-cdk)
+
+---
+
+## Contents
+
+We've provided three example Lambdas to demonstrates the basics of deploying LaunchDarkly within AWS serverless.
+
+1. **getFlagsStarter** – This example uses the LaunchDarkly Node.js server SDK to demonstrate how to initialize the SDK client, get a flag value and, as a base use case, establish separate code paths that can be tested within your Lambda. The example uses an anonymous user context which could be used for randomized experimentation or percentage-based progressive rollouts. For more granular control of targeting for testing, you can pass an environment context or send Cognito user data.
```

Comment:
Consider adding links from the function name to the code.

Reply:
Done